insights/parsers/libssh_config.py

```diff
@@ -0,0 +1,80 @@
+"""
+Libssh configuration files
+==========================
+
+This module provides the paths of configuration file for libssh package
+
+Parsers provided by this module are:
+
+LibsshClientConfig - file ``/etc/libssh/libssh_client.config``
+--------------------------------------------------------------
+
+LibsshServerConfig - file ``/etc/libssh/libssh_server.config``
+--------------------------------------------------------------
+"""
+
+import re
+from insights.specs import Specs
+from insights.core import CommandParser
+from .. import parser, get_active_lines
+from insights.parsers import SkipException
+
+# regex for matching the 'include' directives
+INCLUDE_RE = re.compile(r'^Include\s+\S+')
```

Comment:
`regex` is not the recommended way to parse the simple content if there are other simple methods, like `split` .
Would you please re-implement this logic by using `startswith` or `in` and `split`?